ta/pkcs11/src/pkcs11_attributes.c

```diff
@@ -745,19 +785,22 @@ create_attributes_from_template(struct obj_attrs **out, void *template,
 	if (rc)
 		goto out;
 
-	if (get_attribute(attrs, PKCS11_CKA_LOCAL, NULL, NULL) !=
+	if (get_attribute(temp, PKCS11_CKA_LOCAL, NULL, NULL) !=
```

Comment:
Just a note -- no need to change if you feel current is better:

If you are not interested in attribute's value at all then `get_attribute_ptr(temp, PKCS11_CKA_LOCAL, NULL, NULL)` would be slightly more lightweight but `get_attribute()` also gets the job done.

Reply:
Good idea. I will change this.